docs/source/guides/syntax.md

```diff
@@ -347,7 +347,7 @@ trait ToString {
 }
 ```
 
-Traits aren't allowed to define static methods.
+Traits aren't allowed to define static methods and cannot be used to specify async method interfaces.
```

Comment:
I'd go with this, since "interfaces" is a bit confusing as the term is also used for what are essentially traits without default methods:

```suggestion
Traits can't specify static or async methods.
```